src/app/common/search-select/search-select.component.ts

```diff
@@ -57,7 +57,8 @@ export class SearchSelectComponent implements OnInit {
             this.skillExists = true;
             this.skillCreated = true;
             this.allSkills.push(new Skill(0, this.searchTerm));   
-            setTimeout(()=>{ this.searchTerm = ""; this.skillCreated=false; }, 1100)  
+            setTimeout(()=>{ this.searchTerm = ""; this.skillCreated=false; this.options = this.allSkills; }, 1100);
```

Comment:
I'm not sure exactly what is going on in this section. This might be acceptable, but could you explain what this change is about just to make sure?

Reply:
This auto-resets the skill list box so user can immediately see the added skill.